menu.rb

```diff
@@ -0,0 +1,21 @@
+# How many menu items would the user like to see?
+puts "Welcome to the crazy random menu generator!"
+puts "How many menu items would you like to create?"
+menu_items = gets.chomp.to_i
+
+# Create 3 arrays, 1. food adjectives, 2. food preparation 3. foods (aka rare animals)
+adj 		= ["Melty", "Spicy", "Buttered", "Gourmet", "Fresh", "Hot", "Crunchy", "Sweet", "Savory", 
+			"Marinated", "Chocolatey", "Flaming", "Creamy", "Polka-dotted", "Chunky"]
+prep 		= ["pan-fried", "sauteed", "steamed", "roasted", "baked", "boiled", "broiled", "fried", 
+			"mashed", "breaded", "scorched", "barbecue", "slow-roasted", "braised", "chopped", "raw", 
+			"sous vide", "poached"]
+rare_animals 	= ["panda", "unicorn", "manatee", "wooly mammoth", "bald eagle", "gorilla", "emperor penguin", 
```

Comment:
Moderately disturbed by your rare_animals collection - but the "crazy" factor definitely holds true :+1: